src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts

```diff
@@ -684,4 +666,14 @@ export class UbsAdminEmployeeComponent implements OnInit {
       panelClass: 'admin-cabinet-dialog-container'
     });
   }
+
+  public getExistingCard(filterData) {
```

Comment:
this parameter is not used in the method

Reply:
method was removed